docs/usage/configuration-options.md

```diff
@@ -2688,7 +2688,7 @@ This option is matched against the `currentValue` field of a dependency.
 {
   "packageRules": [
     {
-      "matchPackagePatterns": ["io.github.resilience4j"],
+      "matchPackageNames": ["io.github.resilience4j**"],
```

Comment:
Your migration 
```suggestion
      "matchPackageNames": ["/io\.github\.resilience4j/"],
```
This is not equivalent as your migration expect to start with `io.github.resilience4j`.

Reply:
Since these are examples of how to use `matchCurrentValue` and `matchNewValue`, not `matchPackageNames`, it doesn't actually matter what the value is. So I went with consistency, both with the other values on the same page, and with the most recent changes:

https://github.com/renovatebot/renovate/commit/879f7cc2120f676a0776be90de184e2191fda4e7#diff-be0a39f342380493f17c380bbd694f1f05d18bdd8d7cecf174369084952102deL2788